<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove clashed frames</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Provide here a list of atom ids separated by a comma. The listed atoms will be used to calculate their paiwise distance. If the distance between one atom pair is smaller than the minimum distance this structure is excluded from the trajectory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="plainText">
      <string>name CA and resSeq 1 to 256</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Trajecotry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>stride</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.850000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Min dist [Ang]</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Atom-selection</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpen_trajectory">
   <property name="text">
    <string>Open trajectory</string>
   </property>
  </action>
  <action name="actionSave_clash_free_trajectory">
   <property name="text">
    <string>Save clash free trajectory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen_trajectory</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSave_clash_free_trajectory</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
